FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -166,23 +166,39 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </objectDesc>
                          
                         <bindingDesc>
-                            <binding contemporary="partly"></binding>
+                            <binding contemporary="partly">
                             <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>    
                             <decoNote xml:id="b2" type="SewingStations">4</decoNote>
                             <decoNote xml:id="b3" type="bindingMaterial"><material key="wood"></material></decoNote>
+                            </binding>
                         </bindingDesc>
                        
                     </physDesc>
-                    <history><provenance>Scribe <persName>አርሳንዮስ፡</persName> mentioned in <locus target="coloph1"/> and <locus target="coloph2"></locus></provenance></history>
-                    <msPart xml:id="p1"><msIdentifier><msContents>
-                        <layoutDesc></layoutDesc>
-                        <summary></summary><msContents/><msItem xml:id="p1_i1">
```

Comment:
`<msContents/>` --> `<msContents>` - the tag may only be closed after the content (msItems) has been fully described

check https://github.com/BetaMasaheft/Manuscripts/pull/1978/commits/5782128e5a0d55b56c66615ac017ae2a2f90f456 for a summary of corrections